core/src/main/java/org/apache/iceberg/BaseTableScan.java

```diff
@@ -306,6 +306,6 @@ private Schema lazyColumnProjection() {
   }
 
   private static String formatTimestampMillis(long millis) {
-    return DATE_FORMAT.format(LocalDateTime.ofInstant(Instant.ofEpochMilli(millis), ZoneId.systemDefault()));
```

Comment:
This is a slightly different behavior ... we used to use the System Default and now we are doing UTC?

Reply:
yes, this was an error reported from the newer error-prone checks

> com.palantir.baseline-error-prone uses a newer error-prone version (and does additional checks), so a few changes were necessary to fix JavaTimeSystemDefaultTimeZone / ConsistentOverrides errors